pkg/csi/service/vanilla/controller_test.go

```diff
@@ -286,12 +286,8 @@ func (f *FakeNodeManager) GetAllNodesByVC(ctx context.Context, vcHost string) ([
 }
 
 func (f *FakeAuthManager) GetDatastoreMapForBlockVolumes(ctx context.Context) map[string]*cnsvsphere.DatastoreInfo {
-	datastoreMapForBlockVolumes := make(map[string]*cnsvsphere.DatastoreInfo)
 	fmt.Print("FakeAuthManager: GetDatastoreMapForBlockVolumes")
-	if v := os.Getenv("VSPHERE_DATACENTER"); v != "" {
```

Comment:
Why did we remove this check?

Reply:
We are not setting `VSPHERE_DATACENTER` anywhere so this mock method which is called from filterDatastores() during the unit test, returned an empty response the filter method returned "auth service: no shared datastore found for block volume provisioning".

Reply:
On the current main, even though the fss is enabled, I think the path for this feature wasn't getting executed in the unit tests because https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/common/unittestcommon/utils.go#L44 doesn't have this feature listed so the default value when the fss was checked during the unit test was false.

Reply:
have you verified unit tests after this change?
can you execute `make test` and confirm unit tests are working fine.

Reply:
Yes, unit test is also run as part of the PR tests on github though. They did pass.

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_vsphere-csi-driver/2769/pull-vsphere-csi-driver-unit-test/1751002982792564736